Controllers/TeamController.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Threading.Tasks;
+using CribblyBackend.Models;
+using CribblyBackend.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CribblyBackend.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class TeamController : ControllerBase
+    {
+        private readonly ITeamService TeamService;
+        public TeamController(ITeamService TeamService)
+        {
+            this.TeamService = TeamService;
+        }
+
+        [HttpGet]
+        public async Task<IActionResult> GetById([FromQuery] int id)
+        {
+            var p = await TeamService.GetById(id);
+            if (p != null)
+            {
+                return Ok(await TeamService.GetById(id));
```

Comment:
No need to await this again, just return `p`. It looks like you copied this directly from the Player controller so maybe I need to fix it there too :smile:

Reply:
fixed